### PR TITLE
fix: fixed security-bootstrapper Docker volume init semantics

### PR DESCRIPTION
**Pull request by vli11**
fixes: edgex-go issue [ #3851](https://github.com/edgexfoundry/edgex-go/issues/3851) to remove work-around
related PR from edgex-go: [#4085](https://github.com/edgexfoundry/edgex-go/pull/4085) 
Signed-off-by: Valina Li <valina.li@intel.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)

### CODE REVIEW

File: deployment/helm/templates/edgex-security-bootstrapper/edgex-security-bootstrapper-deployment.yaml [hunk 1]
```diff
@@ -22,19 +22,6 @@ spec:
         org.edgexfoundry.service: {{.Values.edgex.app.bootstrapper}}
     spec:
       automountServiceAccountToken: false
-      initContainers:
```
[the remark this answers is not in the document]
Comment (jim-wang-intel) [reply deployment/helm/templates/edgex-security-bootstrapper/edgex-security-bootstrapper-deployment.yaml:25]: yes, when we tested it we use this local image with others in 2.2.0; i.e. `security-bootstrapper` is local built docker image tag `0.0.0-dev` and others are pulling from `2.2.0` tag

[the remark this answers is not in the document]
Comment (lenny-intel) [reply deployment/helm/templates/edgex-security-bootstrapper/edgex-security-bootstrapper-deployment.yaml:25]: Ok, but this change doesn't change the bootstrapper image. Still uses 2.2.0.

**Comment by jim-wang-intel** [comment]
**Hold** until we have correct docker image deployed in nexus

**Comment by bnevis-i** [comment]
Unholding.  Please merge.

File: deployment/helm/values.yaml [hunk 2]
```diff
@@ -101,11 +101,11 @@ edgex:
       pullPolicy: IfNotPresent
     bootstrapper:
       repository: edgexfoundry/security-bootstrapper
-      tag: "2.2.0"
+      tag: "latest"
```
Comment (bnevis-i) [inline deployment/helm/values.yaml:104]: There is no "edgexfoundry/security-bootstrapper:latest" on Dockerhub.

Please change "respository" to point to the Nexus images.

See https://github.com/edgexfoundry/edgex-compose/blob/main/compose-builder/.env

**Review by bnevis-i — CHANGES_REQUESTED** [review]

**Review by bnevis-i — APPROVED** [review]
Tested. Works fine.

**Pull request merged**